pages/database-management/experimental-features.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Experimental features
+description: Experimental features are new functionalities in Memgraph that carry some risk. They are still in development and may have stability issues.
+---
+
+import { Callout } from 'nextra/components'
+import { Card, Cards } from 'nextra/components'
+
+# Experimental features
+
+These features are actively being developed and may change or be removed in
+future releases. Use them with caution in production environments and only if
+you thoroughly understand their implications and are prepared for potential API
+changes, data loss, service interruption, or data inconsistency. It is highly
+encouraged to thoroughly test the experimental features in your environment.
+
+<Callout>
+
+To enable multiple experimental features in Memgraph, you need to use the
+`--experimental-enabled` flag followed by the name of the feature. Here's how
+you can enable the `high-availability`, `system-replication`, and `text-search`
+features at the same time:
+
+`--experimental-enabled=high-availability,system-replication,text-search`
+
+If you already have a Memgraph instance with an experimental feature enabled,
+and you want to add another one, you will need to restart Memgraph. Ensure that
+all the necessary flags are enabled during the restart.
+
+</Callout>
+
+## Experimental features list
+
+### High availability mode
+- **Description**: This feature enables high availability capabilities in Memgraph Enterprise Edition.
+- **Status**: Experimental. Designed to improve system reliability and uptime, but may still have stability issues.
+- **Usage**: Enable the feature using the flag `--experimental-enabled=high-availability`.
+- **Documentation**: For more details, visit the [High Availability Mode documentation](/database-management/configuration).
+
+### System replication
+- **Description**: This feature enables authentication and multi-tenancy replication in Memgraph Enterprise Edition.
```

Comment:
Avoid using a term to define itself. In this case, "replication" should be referring to headers and followers, along with a mention of the choice between sync and async, and perhaps a mention of horizontal scalability.